production/ksonnet/loki/ingester.libsonnet

```diff
@@ -14,16 +14,18 @@
     container.mixin.readinessProbe.httpGet.withPort($._config.http_listen_port) +
     container.mixin.readinessProbe.withInitialDelaySeconds(15) +
     container.mixin.readinessProbe.withTimeoutSeconds(1) +
-    $.util.resourcesRequests('1', '5Gi') +
-    $.util.resourcesLimits('2', '10Gi'),
+    $.util.resourcesRequests($._config.ingester.CPURequests, $._config.ingester.memoryRequests) +
+    $.util.resourcesLimits($._config.ingester.CPULimits, $._config.ingester.memoryLimits),
 
   local deployment = $.apps.v1.deployment,
 
   local name = 'ingester',
 
   ingester_deployment:
```

Comment:
make sure you are up-to-date with code merged this morning from https://github.com/grafana/loki/pull/2547/files , it does replace ( when boldtb-shipper is enabled ) the `ingester` and `querier` with a statefulset that would need the same changes applied ?

Reply:
yes. I notice when I opened my PR. I will reuse the same variables for both.

Reply:
I updated all new statefulsets.